test/models/tf_ov_model_zoo/run_infer_single.sh

```diff
@@ -19,11 +19,17 @@ INFER_PATTERN=$3
 if [ "${INFER_PATTERN}" == "" ]; then
     echo "Error: expected pattern not specified!" && exit 1
 fi
+BENCHMARK=$4 # YES or NO
+if [ "${BENCHMARK}" == "" ]; then
+    echo "Error: benchmark flag (YES/NO) not specified!" && exit 1
+fi
 
-echo MODEL=$MODEL IMAGE=$IMAGE INFER_PATTERN=$INFER_PATTERN
+echo MODEL=$MODEL IMAGE=$IMAGE INFER_PATTERN=$INFER_PATTERN BENCHMARK=$BENCHMARK
 
 REPO=https://gitlab.devtools.intel.com/mcavus/tensorflow_openvino_models_public
-COMMIT=14eff8ce # 2020-Dec-02
+COMMIT=d12f2d57 # 2021-Jan-06
```

Comment:
Is this necessary?

Reply:
Actually, it's better to fix this than use the latest. Never mind my question above.

Reply:
Yes, we don't want a moving baseline from other sources, as it might break our CI. With known baseline, we know it works.